### ISSUE TITLE
Double-dash occurs at the hyphen of a compound

### ISSUE THREAD

**Issue by ppasedach** [maieul/ledmac#326]
The following code gives me a weird double dash when the line break occurs in the middle of "half-verse"

```
\documentclass{memoir}
\usepackage{fontspec}
\usepackage{eledmac}
\footparagraph{A}
\numberonlyfirstinline[A]
\begin{document}
\beginnumbering
\pstart

\edtext{}{\Afootnote[nosep]{\textit{māṣa'}] a bare stem form is used in the
sense of genitive; \textit{muṅgamasūrayo'} for \textit{mudgamasūrayoḥ}. All
genitives in this xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxx xxxxxxxx half-verse are used in the sense of accusative.}}%  


\pend
\endnumbering

\end{document}
```

![auswahl_009](https://cloud.githubusercontent.com/assets/11517856/8397950/0f12222a-1dde-11e5-8602-3ec6107a13be.png)

**Comment by maieul** [comment]
hum, it will be quite complex to find the bug and solve it. 

However, why do you use
```\Afootnote[nosep]{\textit{māṣa'}]```

that is a non-sense

**Comment by ppasedach** [comment]
I understand this is not the way eledmac is supposed to be used, but it seemed to be the easiest way to "upgrade" an article with simple footnotes for the apparatus of quite a few pages of edition, written by somebody else, to use eledmac for that. Changing the footnote to  ```\edtext{\textit{māṣa'}}{\Afootnote{ a bare stem form is used in the sense of genitive; \textit{muṅgamasūrayo'} for \textit{mudgamasūrayoḥ}. All genitives in this xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx xxxxxx xxxxxxxx half-verse are used in the sense of accusative.}}%```, as, if I get it right, it should actually be done, also triggers the bug, so it doesn't seem to be caused by my admittedly odd way of using eledmac. Anyway I can do a workaround with ```\discretionary{half-}{verse}{half-verse}```, that works for the time being.

**Comment by maieul** [comment]
yes, I didn't think you odd way of using eledmac was the cause. I think the problem is the way eledmac works for paragraphed footnote. But for now, I have no answer. I think I should ask to expert with a better MWE (i.e. a MWE directly in TeX, not with eledmac)
